pydash/arrays.py

```diff
@@ -1813,6 +1912,9 @@ def iterunique(array, comparator=None, iteratee=None):
             seen.append(cmp_item)
 
 
+iterunion = iterunique
```

Comment:
There isn't really a reason to alias. Can just use `iterunique` directly where needed.